GoogleSheetI18n/Core/I18nLocalStore.cs

```diff
@@ -64,5 +64,20 @@ private string GetLocalStoreKey(string spreadsheetId, string sheetName)
         {
             return $"{spreadsheetId}.{sheetName}.json";
         }
+
+        public async Task<IList<I18nSheet>> GetSheets(string spreadsheetId)
+        {
+            var sheets = new List<I18nSheet>();
+            DirectoryInfo dir = new DirectoryInfo(_localStoreFolderPath);
+            FileInfo[] filesInDir = dir.GetFiles("*" + spreadsheetId + "*.*");
```

Comment:
You can safely specify `json` extension here.

Reply:
used ### json